app/views/layouts/design_system.html.erb

```diff
@@ -15,7 +15,7 @@
            environment: environment,
            browser_title: ("Error: " if yield(:error_summary).present?).to_s + sanitized_title do %>
 
-  <div data-module="GA4PageViewTracking" data-attributes='<%= track_analytics_data_on_load(sanitized_title) %>'></div>
+  <div data-module="GA4PageViewTracking GA4PasteTracker" data-attributes='<%= track_analytics_data_on_load(sanitized_title) %>'></div>
```

Comment:
Nitpick: one is `Tracking`, one is `Tracker` - should they be consistent?

Reply:
`govuk_publishing_components` uses "tracker" throughout so I think that it's best to stick to that
https://github.com/alphagov/govuk_publishing_components/tree/main/app/assets/javascripts/govuk_publishing_components/analytics-ga4